### PR TITLE
test: stabilize e2e navigation with Playwright

### PR DESCRIPTION
**Pull request by fbasso**
Added some waitForSelector to stabilize the navigation

### CODE REVIEW

File: e2e-app/src/app/tools.pw-po.ts [hunk 1]
```diff
@@ -94,12 +94,16 @@ export const expectFocused = async(selector, message) => {
  */
 let hasBeenLoaded = false;
 export const openUrl = async(url: string) => {
+  const p = page();
```
Comment (maxokorokov) [inline e2e-app/src/app/tools.pw-po.ts:97]: Oh, I actually don't like this at all :D

**Review by maxokorokov — COMMENTED** [review]
I don't understand the fix, to be honest...

Specifically why we're waiting for `ng-component` in detached state...

**Comment by fbasso** [comment]
It is not a fix of an open issue, but I think it's better to be sure that the navigation produce the right effect before continuing more actions.

  - waitForSelector for 'app-root' after the goto : ensure that the page is well loaded,
  - waitForSelector for 'ng-component', with the detached : be sure that the previous test is unloaded (the menu page is ready then)
  - waitForSelector for 'ng-component' : be sure that the component page is ready.

**Review by maxokorokov — APPROVED** [review]
LGTM, please merge when green!

**Pull request merged**